.dockerignore

```diff
@@ -2,8 +2,7 @@
 .gitignore
 AUTHORS
 CONTRIBUTING.md
-docker-compose.yml
 LICENSE
 node_modules/
 README.md
-flutter/
+./flutter/
```

Comment:
I believe this should be `/flutter/`

Reply:
Good point. Thanks!

Revisiting this, we don't even need the entry anymore anyway since we don't clone Flutter in the build process anymore.